features/src/main/feature/camel-features.xml

```diff
@@ -652,6 +652,14 @@
         <feature version='${project.version}'>camel-cxf</feature>
         <bundle>mvn:org.apache.camel.karaf/camel-cxf-spring-all/${project.version}</bundle>
     </feature>
+    <feature name='camel-datasonnet' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-cxf</feature>
```

Comment:
Why do you refer to `camel-cxf` here? I don't see any relationship with CXF

Reply:
wrong copy paste from the feature above, replaced by camel-core